docs/docs/06-tools/01-cli.md

```diff
@@ -222,3 +222,24 @@ $ wing pack
 This will compile your current Wing directory, and bundle it as a tarball that can be published to [GitHub packages](https://github.com/features/packages) or [npm](https://www.npmjs.com/).
 
 See [Libraries](../05-libraries.md) for more details on packaging and consuming Wing libraries.
+
+## Environment Variables
+
+Wing uses `dotenv` to load additional environment variables from files in your current working directory. These environment variables can be accessed in Wing code using `util.env` and `util.tryEnv`.
```

Comment:
Should we maybe adjust the documentation to avoid assuming prior knowledge about the npm library? I see the phrase "dotenv" used but it's not clear to me if it's some well-defined standard (if it is then I'm cool with just linking that). It might be a better user experience if we explain how it works and how a `.env` file looks. The other upside is it keeps our API promise more stable in case we change to using a library other than `dotenv`, or rewrite the CLI in Rust etc.

```suggestion
For development and testing, Wing can automatically read environment variables from `.env` files in your current working directory. These environment variables can be accessed in Wing code using `util.env` and `util.tryEnv`.
```